example/index.ts

```diff
@@ -13,20 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { inspect } from 'util';
 
-'use strict';
+import { list } from '..';
 
-const drivelist = require('..');
-const util = require('util');
+async function main() {
+	try {
+		const drives = await list();
+	} catch (error) {
+		console.error(error);
+		process.exitCode = 1;
```

Comment:
Any reason you moved away from `process.exit(1)`?  This new form doesn't guarantee the process will exit on an error (if there's anything in the background listening or similar)

Reply:
There should be nothing left listening in the background (or it's a bug).
I've had issues in projects using `process.exit` before so I try to avoid it.